code/wdb/wdbstandard.q

```diff
@@ -0,0 +1,6 @@
+\d .wdb
+startup:{[f]
+	f[]; /-Call the original startup function
+	@[`.;`upd;:;{[t;x]}]; 
+	};[startup]
```

Comment:
I don't think this will work? that `;` will mean this isn't passed in to the function